.travis.yml

```diff
@@ -42,7 +42,6 @@ env:
     - KNAPSACK_PRO_CI_NODE_TOTAL=3
     - COVERAGE_REPORTS_TOTAL=4
     - FOREM_OWNER_SECRET="secret" # test secret so e2e tests can run properly.
-    - DISABLE_SPRING=true
```

Comment:
I moved my HEAD back to the initial point when I cloned but when this PR was initially created it had all these changes because I took pull lately from main.

Reply:
I moved my HEAD back to the initial point when I cloned but when this PR was initially created it had all these changes because I took pull lately from main.

Reply:
I moved my HEAD back to the initial point when I cloned but when this PR was initially created it had all these changes because I took pull lately from main.

Reply:
I guess this PR should be closed as so many unwanted changes have been introduced I don't know how.

Reply:
@papuruth Up to you! Feel free to make a new PR with a cleaner git history, or try to fix this branch. I personally would make a new PR at this point. 😅